main.py

```diff
@@ -74,18 +90,47 @@ def create_histogram_fig(df, DAYS):
         df,
         x="Date",
         y="Cyclists_Injured",
+        color_discrete_map=BOROUGH_COLORS,
         color="Borough",
+        hover_data={
+            "Borough": True,
+            "crash_date_str": True,
+            "Cyclists_Injured": True,
+        },
+        labels={
+            "crash_date_str": "",
+            "borough": "Borough",
+            "Cyclists_Injured": "Borough Cyclists Injured",
+        },
         nbins=30,
-        title="Recent Cyclist Injuries By Borough",
-        labels={"Date": "Day", "Cyclists_Injured": "Cyclist Injuries"},
+        title="Cyclist Injuries By Day",
         height=400,
     )
     histogram_fig.update_layout(
-        margin=dict(l=40, r=20, t=30, b=5),
+        margin=dict(l=80, r=20, t=30, b=5),
         bargap=0.1,
         paper_bgcolor="rgba(0,0,0,0)",
         plot_bgcolor="rgba(0,0,0,0)",
+        title={
```

Comment:
This title dictionary looks a lot like the one defined above. Should we just define the title as in `constants.py` and import it?

Maybe there is a way to cut down on repeated code (always a good thing to think about as this future proofs your work)